src/patient-registration/patient-registration.component.tsx

```diff
@@ -533,7 +533,7 @@ export const PatientRegistration: React.FC = () => {
               <Row>
                 <Column lg={2} md={2} sm={1}>
                   <div className={styles.fixedPosition}>
-                    <h4>{existingPatient ? 'Edit' : 'Create New'} Patient</h4>
+                    <h4 style={{ marginTop: '10px' }}>{existingPatient ? 'Edit' : 'Create New'} Patient</h4>
```

Comment:
Not sure if this is the right approach.

1. I don't like px. They don't scale.
2. Is this from the original design? Doesn't it rather talk ems?
3. I'd rather see one or the classes adopted / improved than a custom style inserted in a workaround-like fashion.